### PR TITLE
Fix typos in C5 and C6

### PR DESCRIPTION
**Pull request by iod-ine**
Minor typos in chapters 5 and 6.

### CODE REVIEW

File: 05-geometry-operations.Rmd [hunk 1]
```diff
@@ -559,7 +559,7 @@ A variety of georeferencing techniques exist, including:
 
 - Georectification based on known [ground control points](https://www.qgistutorials.com/en/docs/3/georeferencing_basics.html);
 - Orthorectification, which also accounts for local topography
-- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors the process of aligning one image with another (in terms of coordinate system, and resolution)
+- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors by aligning one image with another (in terms of coordinate system, and resolution)
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:562]: Definite improvement. While reading this `, and` jumps out at me and I suggest removing the superfluous punctuation, e.g. updating this to read:

```
- Image [registration](https://en.wikipedia.org/wiki/Image_registration) is used to combine images of the same thing but shot from different sensors by aligning one image with another (in terms of coordinate system and resolution)
```

Sound reasonable?

File: 05-geometry-operations.Rmd [hunk 2]
```diff
@@ -829,7 +829,7 @@ We will use two objects to illustrate raster cropping:
 - A vector (`sf`) object `zion` representing Zion National Park.
 
 Both target and cropping objects must have the same projection.
-The following code chunk therefore not only reads the datasets from the **spDataLarge** package9 installed in Chapter \@ref(spatial-class)), it also reprojects `zion` (see Section \@ref(reproj-geo-data) for more on reprojection):
+The following code chunk therefore not only reads the datasets from the **spDataLarge** package (installed in Chapter \@ref(spatial-class)), it also reprojects `zion` (see Section \@ref(reproj-geo-data) for more on reprojection):
```
Comment (Robinlovelace) [inline 05-geometry-operations.Rmd:832]: :+1:

File: 06-reproj.Rmd [hunk 3]
```diff
@@ -140,7 +140,7 @@ st_distance(london_geo, london_proj)
 
 To make the `london` and `london_proj` objects geographically comparable one of them must be transformed into the CRS of the other.
 But which CRS to use?
-The answer is usually 'to the projected CRS', which in this case is the British National Grid (EPSG:27700):
+The answer is usually 'the projected CRS', which in this case is the British National Grid (EPSG:27700):
```
Comment (Robinlovelace) [inline 06-reproj.Rmd:143]: :+1: and +1 for reducing the word count, less is is more in this case!

**Review by Robinlovelace — APPROVED** [review]
Definite improvement so approved but maybe worth one more tweak on line 562.

File: 05-geometry-operations.Rmd (hunk 1 above)
Comment (iod-ine) [reply 05-geometry-operations.Rmd:562]: Absolutely! I'll update it :)

**Comment by Robinlovelace** [comment]
Thanks @iod-ine !

**Pull request merged**